### ISSUE TITLE
Issue with creating access with 'create-only' level.

### ISSUE THREAD

**Issue by SerixeN** [pryv/open-pryv.io#7]
Hello everyone.

I'm writing here, because there is issue with creation access with 'create-only' level. For others levels it works fine, but for 'create-only' I get next error:

```
{
  'error': {
    'id': 'invalid-parameters-format',
    'message': "The parameters' format is invalid.",
    'data': [
      {
        'code': 'ANY_OF_MISSING',
        'params': [
          
        ],
        'message': "Data does not match any schemas from 'anyOf'",
        'path': '#/',
        'schemaId': 'pryv.core.access-create',
        'inner': [
          {
            'code': 'OBJECT_ADDITIONAL_PROPERTIES',
            'params': [
              [
                'expireAfter'
              ]
            ],
            'message': 'Additional properties not allowed: expireAfter',
            'path': '#/'
          },
          {
            'code': 'ONE_OF_MISSING',
            'params': [
              
            ],
            'message': "Data does not match any schemas from 'oneOf'",
            'path': '#/permissions/0',
            'inner': [
              {
                'code': 'ENUM_MISMATCH',
                'params': [
                  'create-only'
                ],
                'message': 'No enum match for: create-only',
                'path': '#/permissions/0/level',
                'schemaId': 'streamPermission'
              },
              {
                'code': 'OBJECT_MISSING_REQUIRED_PROPERTY',
                'params': [
                  'tag'
                ],
                'message': 'Missing required property: tag',
                'path': '#/permissions/0',
                'schemaId': 'tagPermission'
              }
            ]
          },
          {
            'code': 'ENUM_MISMATCH',
            'params': [
              'app'
            ],
            'message': 'No enum match for: app',
            'path': '#/type'
          }
        ]
      }
    ]
  },
  'meta': {
    'apiVersion': '1.4.23',
    'serverTime': 1597152581.771,
    'serial': '2019120401'
  }
}
```
Please let me know how it can be fixed. Thanks.

**Comment by kebetsi** [comment]
Hi @SerixeN,

can you provide the request body that you are using for your request?

**Comment by SerixeN** [comment]
Yes, @kebetsi here is my request body
```
{
        "name": "test",
        "permissions": [
            {
                "streamId": "*",
                "level": "create-only"
            }
        ],
        "type": "app",
        "expireAfter": 4102444800 - time.time()
}
```

**Comment by kebetsi** [comment]
The `create-only` level was released in Pryv.io 1.4.33, which is not available in the 1.4.23 that you are using. 

Please contact us through to obtain the latest configuration at support@pryvhelp.zendesk.com or open on a post in our Help Center https://support.pryv.com/hc/en-us/community/topics.
